core/field_colour.js

```diff
@@ -120,15 +120,6 @@ Blockly.FieldColour.prototype.SERIALIZABLE = true;
  */
 Blockly.FieldColour.prototype.CURSOR = 'default';
 
-/**
- * Used to tell if the field needs to be rendered the next time the block is
- * rendered. Colour fields are statically sized, and only need to be
- * rendered at initialization.
- * @type {boolean}
- * @protected
- */
-Blockly.FieldColour.prototype.isDirty_ = false;
```

Comment:
Did you decide to remove the static rendering just for readability reasons?

Honestly I don't really mind b/c I don't think it was that much of an efficiency improvement. I'm just curious.

Reply:
I think you might be onto something in the case of field_colour. Will double check now, but for field_checkbox we needed to re-render as the checkmark's font size may have changed so removed the isDirty_ flag as it's no longer a static field.

Reply:
Revert field colour to a static field :)